hooks/useIpfsGet.ts

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import axios, { AxiosError } from 'axios'
+import { useToast } from '@/hooks/useToast'
+
+export const useIpfsGet = () => {  
```

Comment:
useTokenURIMetaData
or 
useTokenURIJSON

Reply:
I thought that we might want a verb after "use", since this does perform an action. How about:

`useFetchTokenMetadata`

Reply:
Both names (this and the commend below) changed in https://github.com/henkaku-center/omise-interface/pull/80/commits/cef47f1d12f1f5ce416b485ed832b6c64c4abc7c